### PR TITLE
footer updated

### PR DESCRIPTION
**Pull request by shivam0110**

### CODE REVIEW

**Review by prathmeshkhandelwal1 — CHANGES_REQUESTED** [review]
@shivam0110 please squash your commits.

**Comment by shivam0110** [comment]
Done

**Comment by shivam0110** [comment]
done most prolly

**Review by prathmeshkhandelwal1 — CHANGES_REQUESTED** [review]
Screenshot (177)

there should be much gap bw logo and footer start according to design,
Line above the `Developed by coding club` section has gaps in their horizontal length according to design,

**Comment by prathmeshkhandelwal1** [comment]
and also there is horizontal scroll appearing bcoz of this PR, maybe this can fix that `box-sizing: border-box`

File: src/Component/Footer/footcss.css [hunk 1]
```diff
@@ -77,7 +87,7 @@ footer {
 .link,
```
Comment (AmanRaj1608) [inline src/Component/Footer/footcss.css:92]: We should have a hover effect
just add a simple underline that's enough

**Review by AmanRaj1608 — COMMENTED** [review]
@pAditya198 thoughts?

File: src/Component/Footer/footcss.css (hunk 1 above)
Comment (shivam0110) [reply src/Component/Footer/footcss.css:92]: done

**Pull request merged**